src/components/Table/GridTable.tsx

```diff
@@ -636,9 +636,12 @@ export function calcVirtualGridColumns(columns: GridColumn<any>[], firstLastColu
     { claimedPercentages: 0, claimedPixels: 0, totalFr: 0 },
```

Comment:
It might be too cute, but you could do `claimedPixels: firstLastColumnWidth ? firstLastColumnWidth * 2 : 0` to sneak it in here and avoid the separate `totalClaimedPixels` adjustment later